libs/shared/src/entities/messages/messages.interface.ts

```diff
@@ -20,5 +20,13 @@ export interface IMessage {
       url?: string;
     };
   };
+  _feedId: string;
   payload: Record<string, unknown>;
 }
+
+export interface IFeedEntity {
+  _id: string;
+  name: string;
+  _environmentId: string;
+  _organizationId: string;
```

Comment:
Due to the reason, it is related to another entity scheme, what do you think about moving it to a dedicated file?